decidim-proposals/lib/decidim/proposals/engine.rb

```diff
@@ -41,6 +41,24 @@ class Engine < ::Rails::Engine
         get "/", to: redirect("proposals", status: 301)
       end
 
+      initializer "decidim_proposals.register_icons" do
+        Decidim.icons.register(name: "Decidim::Proposals::CollaborativeDraft", icon: "draft-line", category: "activity",
+                               description: "Collaborative draft", engine: :proposals)
+        Decidim.icons.register(name: "Decidim::Proposals::Proposal", icon: "chat-new-line", category: "activity",
+                               description: "Proposal", engine: :proposals)
+        Decidim.icons.register(name: "participatory_texts_item", icon: "bookmark-line", description: "Index item", category: "participatory_texts",
+                               engine: :proposals)
+
+        Decidim.icons.register(name: "scan-line", icon: "scan-line", category: "system", description: "", engine: :proposals)
+        Decidim.icons.register(name: "edit-2-line", icon: "edit-2-line",
+                               category: "actions", description: "Edit icon for Collaborative Drafts", engine: :proposals)
```

Comment:
Let's merge the "action" and "actions" categories please
```suggestion
                               category: "action", description: "Edit icon for Collaborative Drafts", engine: :proposals)
```